pybandits/actions_manager.py

```
@@ -237,7 +237,9 @@ def _validate_update_params(
                 q is not None for a, q in zip(actions, quantities) if isinstance(self.actions[a], QuantitativeModel)
             ):
                 raise ValueError("Quantitative actions require defined quantities.")
-            if not all(q is None for a, q in zip(actions, quantities) if isinstance(self.actions[a], (Model, ModelMO))):
+            if not all(
+                q is None for a, q in zip(actions, quantities) if not isinstance(self.actions[a], QuantitativeModel)
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)